string chain
bool balance

float64[] eff_pose_current
float64[] initial_thetas
float64[] eff_pose_reference
float64[] balance_rotation
---
float64[] thetas


